main.go

```diff
@@ -400,3 +412,30 @@ func runScorecardSettings(githubEventName, scorecardPolicyFile, scorecardResults
 	}
 	return &result, nil
 }
+
+func signScorecardResult(scorecardResultsFile string) error {
+	err := os.Setenv("COSIGN_EXPERIMENTAL", "true")
+	if err != nil {
+		return fmt.Errorf("error setting COSIGN_EXPERIMENTAL env var: %w", err)
+	}
+	ctx := context.Background()
+
+	// Prepare settings for SignBlobCmd.
+	keyOpts := sign.KeyOpts{
+		FulcioURL:    fulcioURL,                    // Signing certificate provider.
+		RekorURL:     rekorURL,                     // Transparency log.
+		OIDCIssuer:   options.DefaultOIDCIssuerURL, // OIDC provider to get ID token to auth for Fulcio.
+		OIDCClientID: "sigstore",
+	}
+	regOpts := options.RegistryOptions{} // Not necessary so we leave blank.
+
+	// This command will use the provided OIDCIssuer to authenticate into Fulcio, which will generate the
+	// signing certificate on the scorecard result. This attestation is then uploaded to the Rekor transparency log.
+	// The output bytes (signature) and certificate are discarded since verification can be done with just the payload.
+	_, err = sign.SignBlobCmd(ctx, keyOpts, regOpts, scorecardResultsFile, true, "", "", time.Minute)
+	if err != nil {
```

Comment:
nit: you could also check error on the same line since the other rval isn't used